src/schema/v2/partner.ts

```diff
@@ -183,6 +183,35 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
         }),
         resolve: artist => artist,
       },
+      cities: {
+        description: "A list of the partners unique city locations",
+        type: new GraphQLList(GraphQLString),
+        args: {
+          size: {
+            type: GraphQLInt,
+            defaultValue: 25,
+          },
+        },
+        resolve: ({ id }, options, { partnerLocationsLoader }) => {
+          return partnerLocationsLoader(id, options)
+            .then(locations => {
+              const locationCities = locations.body.map(location => {
+                return location.city
+              })
+              const filteredForDuplicatesAndBlanks = locationCities.filter(
+                (city, pos) => {
+                  return (
+                    city &&
+                    locationCities.indexOf(city) === pos &&
+                    city.length > 0
+                  )
+                }
+              )
+              return filteredForDuplicatesAndBlanks
+            })
+            .catch(err => console.log(err))
```

Comment:
Perhaps better to let these errors bubble up?

Reply:
👍